### DISCUSSION TITLE
PCL为啥会被Denfender是为病毒呢

### DISCUSSION THREAD

**Discussion by ghost** [Meloong-Git/PCL#3193] [讨论]
如题

**Comment by MoYuan-CN** [comment]
这是由于 PCL2 没有购买数字签名导致的，开发者没有任何办法解决这一问题。
你可以将 PCL2 添加到防火墙白名单，这样至少不会被删……
只要你是从官方渠道下载的，或者使用 PCL2 内置更新来更新的，都不是病毒，是误报……

—— LTCatt in #56

**Reply by hy1stax** [reply]
看来微软意思是还是让开发桌面应用的人的买他们的签名U盘了。。。

**Reply by allMagicNB** [reply]
别瞎说，现在有 Liu Guoqing 的帮助下，给正式版折腾上了，但是快照版因为更新时会在程序内部留下信息破坏签名，签了也没啥用

**Reply by hy1stax** [reply]
还有这样的，那只能手动白名单了...

**Comment by allMagicNB** [comment]
最近像是有人恶意举报……？微软那边也不受理
其实主要是VMProtect套壳的程序都会报毒，一般都是病毒作者为了不让人反编译而加壳的……

**Comment by WorldHim** [comment]
标题是 `Defender` 吧

**Reply by allMagicNB** [reply]
确实，多打了一个“n”
